libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/backend/loadGranteeOptionsPromise.ts

```diff
@@ -22,11 +23,21 @@ const createErrorOption = (intl: IntlShape): ISelectErrorOption[] => {
     ];
 };
 
+const matchAllGroupQueryString = (query: string, allGroupLabel: string): boolean => {
+    return allGroupLabel.toLocaleLowerCase().indexOf(query.toLocaleLowerCase()) > -1;
```

Comment:
Imho you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes instead of `indexOf > -1`

Reply:
I will keep as it is this construction is used in backend service so I would like to have it same.